feedi/sources/rss.py

```diff
@@ -87,7 +93,10 @@ def _matches(entry, filters):
         return True
 
     def parse_title(self, entry):
-        return entry['title']
+        if entry.has_key('title'):  # Not required by RSS spec
+            return entry['title']
+        else:  # Return first 10 words of description
+            return strip_html(' '.join(entry['description'].split()[:10]))
```

Comment:
I think it's better to use `summary` instead of `description` here. iiuc both would have the same value in most cases, but summary is the "canonical" one (and the one I've been referencing in the rest of the code) 
https://feedparser.readthedocs.io/en/latest/reference-entry-summary.html

Reply:
The feeds I was looking at don't have a `<summary>` field. That's another patch for Mastodon's rss generator. Should I stall this until I tackle the lackluster generation end?